services/docker-compose.yml

```diff
@@ -154,6 +154,9 @@ services:
     networks:
       - default
       - computational_services_subnet
+    configs:
```

Comment:
hmm this could result in a bad merge. check master please.
also the auto-deployer cannot cope with configs at the moment.